_blinka/orange_pi_lite.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: download
+board_id: "orange_pi_lite
+title: "Orange Pi Lite Download"
+name: "Orange Pi Lite"
+manufacturer: "Shenzhen Xunlong Software CO.,Limited"
+board_url: "http://www.orangepi.org/orangepilite/"
+board_image: "orange_pi_lite.png"
+download_instructions: "https://learn.adafruit.com/circuitpython-on-orangepi-linux/circuitpython-orangepi"
+downloads_display: true
+blinka: true
+features:
+  - Wi-Fi
+  - Bluetooth/BLE
+  - HDMI
+  - 40-pin GPIO
+---
+
+What’s Orange Pi PC？It’s an open-source single-board computer. It can run Android 4.4, Ubuntu, Debian Image. It uses the AllWinner H3 SoC, and has 512MB DDR3 SDRAM.
```

Comment:
We should probably change this to Orange Pi Lite. Just grab the info from http://www.orangepi.org/orangepilite/ and format it.

Reply:
Ups, my bad. I have changed this as per your request.